### ISSUE TITLE
Flag `if A { B } else { B }`

### ISSUE THREAD

**Issue by filiph** [dart-lang/linter#169]
Common copy-paste error:

```dart
var variable1;
if (A) {
  variable1 = new SomeLongCopyPasteThing();
} else {
  variable1 = new SomeLongCopyPasteThing();
}
```

Should show something like: `Hint: the "else" part is the same as the "then" part.`. Should ignore comments.
